@here/harp-mapview/lib/Tile.ts

```diff
@@ -42,7 +42,8 @@ function decodedTileHasGeometry(decodedTile: DecodedTile) {
         decodedTile.geometries.length ||
         (decodedTile.poiGeometries !== undefined && decodedTile.poiGeometries.length) ||
         (decodedTile.textGeometries !== undefined && decodedTile.textGeometries.length) ||
-        (decodedTile.textPathGeometries !== undefined && decodedTile.textPathGeometries.length)
+        (decodedTile.textPathGeometries !== undefined && decodedTile.textPathGeometries.length) ||
```

Comment:
Did you test this with a `Theme` that only has your new technique? Most likely it will not work b/c there is some weird handling of tiles that don't have geometries. I fixed that with:
https://github.com/heremaps/harp.gl/pull/902

Reply:
Yes, see: https://gerrit.it.here.com/#/c/1232943/

I saw your fix, but didn't understand what the problem was unfortunately.